test/danetest.c

```diff
@@ -22,12 +22,16 @@
 #ifndef OPENSSL_NO_ENGINE
 #include <openssl/engine.h>
 #endif
+#include "testutil.h"
+#include "test_main_custom.h"
 
-#include "../e_os.h"
+#include "e_os.h"
```

Comment:
It happens when there's a sub-inclusion with a relative path to another directory...  so this particular one doesn't actually break, but we might as well be consistent...